db/__tests__/seed.test.js

```diff
@@ -184,6 +184,59 @@ test('GET 200: responds with array of article objects in descending order', () =
         const { articles } = body
         expect(articles).toBeSortedBy('created_at',{ descending: true })
     })
+}) // topic, which filters the articles by the topic value specified in the query. If the query is omitted, the endpoint should respond with all articles.
+test('GET 200: responds with array of article objects matching the topic query', () => {
+    return request(app)
+    .get('/api/articles?topic=cats')
+    .expect(200)
+    .then(({ body }) => {
+        const { articles } = body
+        articles.forEach((article) => {
+            expect(article).toMatchObject(
+                {
+                    article_id: expect.any(Number),
```

Comment:
It _might_ have been better to query a topic that has more articles associated with it, so that we can assert against a response with more than one response and feel a bit more confident this has done its job.